crates/protocol/build.rs

```diff
@@ -1,7 +1,7 @@
 use std::{env, io::Result, path::PathBuf};
 
 fn main() -> Result<()> {
-    let warg_proto = PathBuf::from("src/warg.proto");
+    let warg_proto = PathBuf::from("src/warg/warg.proto");
```

Comment:
We should probably take this opportunity to move it to a top-level directory rather than under `src/`, as that's where I expect Rust source to live, not protobuf definitions.

Perhaps `crates/protocol/warg/warg.proto`?

Reply:
Cool, yeah that seems good.

Reply:
I actually want to take this a step further. If the point of this is to make the `.proto` accessible outside the Rust crate to other consumers, then it probably shouldn't be in the `crates` directory at all. We should probably put this file in some root folder with the spec when it's written.

Reply:
That's great -- I did want to try proposing how it could be more cross-platform usable -- so I'll just drop the minor fix steps that were just nice to haves, and make a PR that would have all steps to discuss.